pylearn2/space/__init__.py

```diff
@@ -1998,7 +2235,8 @@ def recursive_shared(batch):
     @functools.wraps(Space._format_as_impl)
     def _format_as_impl(self, is_numeric, batch, space):
         """
-        Supports formatting to a single VectorSpace, or to a CompositeSpace.
+        Supports undo formatting to a single VectorSpace,or to a
```

Comment:
This change should probably be reverted, this is the docstring of `_format_as_impl`, not `_undo_format_as_impl`